### PR TITLE
Install dlls to correct location

### PR DESCRIPTION
**Pull request by ghost**
Usally when consuming external libraries you have 3 folders. 

1. Include (contains public headers)
2. Lib (contains the compiled libs)
3. Bin (contains the compiled and linked binaries)

At the moment during cmake install the dll is installed to the lib folder which breaks external tools like conan (C/C++ Package Manager).
I'd like to install the dll to the correct location. :)

### CODE REVIEW

**Review by kisvegabor — APPROVED** [review]

File: examples/get_started/lv_example_get_started_2.c [hunk 1]
```diff
@@ -31,4 +31,3 @@ void lv_example_get_started_2(void)
 }
 
 #endif
-
```
Comment (FASTSHIFT) [inline examples/get_started/lv_example_get_started_2.c:34]: Why delete the last line of the file? If it was deleted accidentally, please keep it.

Comment (kisvegabor) [reply examples/get_started/lv_example_get_started_2.c:34]: I though it's just a rebase issue as some empty lines were removed in an other PR. However rebasing locally  haven't resulted in any changes. 

Anyway, I agree to not remove the last empty lines in the PR.

Comment (ghost) [reply examples/get_started/lv_example_get_started_2.c:34]: I did just run the scripts :P
The only actual change i did was in the CMake File :D

**Review by XuNeo — APPROVED** [review]
The original file has two empty lines, thus code format will remove one.

**Comment by kisvegabor** [comment]
Actually it seems there no empty line at all e.g. [here](https://github.com/lvgl/lvgl/blob/master/examples/widgets/menu/lv_example_menu_5.c#L202).

**Comment by XuNeo** [comment]
I was wrong, the empty line should not be removed!

**Comment by kisvegabor** [comment]
It's strange for me too. I see one empty line in Eclipse, see no empty line on GitHub, and I don't what the code formatter thinks.

**Comment by kisvegabor** [comment]
@FASTSHIFT it was early to merge it. Those line changes should have been removed. 

Anyway, it doesn't matter much (I hope).

**Comment by FASTSHIFT** [comment]
I tried to pull the latest code, the blank lines are still in their original positions. I guess it is a bug displayed on github.

**Pull request merged**